corehq/apps/hqcase/management/commands/ptop_fast_reindexer.py

```diff
@@ -159,13 +166,35 @@ def load_from_view(self):
             fout.write(str(current_db_seq))
 
         # Load entire view to disk
+        if self.sort_key:
+            self._load_from_view_and_sort()
+        else:
+            self._load_from_view()
+        self.log("View and sequence written to disk: %s" % datetime.utcnow().isoformat())
+
+    def _load_from_view_and_sort(self):
+        dump_filename = self.get_dump_filename()
+        dump_presort_filename = dump_filename + '.presort.tmp'
+        self.log('Writing dump file to disk: {}, starting at {}'.format(
+            dump_presort_filename, datetime.utcnow().isoformat()))
+        with open(dump_presort_filename, 'w') as fout:
+            for row in self.full_couch_view_iter():
+                sort_key = self.sort_key(row)
+                if self.sort_key_include_docs and not self.pillow_class.include_docs:
+                    del row['doc']
+                fout.write('{}\t{}\n'.format(sort_key, json.dumps(row)))
```

Comment:
if the sort key ends up with a tab in it for some reason we'd have a problem. maybe worth converting tabs to spaces in the sort_key?